ugvc/reports/bqsr_train_report.ipynb

```diff
@@ -0,0 +1,971 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "usage = \"\"\"Run with papermill:\n",
+    "     \n",
+    "papermill BQSR_report.ipynb output_BQSR_report.ipynb \\\n",
+    "    -p report_name <> \\\n",
+    "    -p model_file <> \\\n",
+    "    -p X_file <> \\\n",
+    "    -p y_file <> \\\n",
+    "    -p params_file <>\n",
+    "\n",
+    "Then convert to html\n",
+    "\n",
+    "jupyter nbconvert --to html output_BQSR_report.ipynb --template classic --no-input --output BQSR_report.html\"\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "df91b0b8-6bdf-4a98-8bd7-b94bdac764b3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import os\n",
+    "from IPython.display import Image, HTML\n",
+    "import joblib\n",
+    "import json\n",
+    "import matplotlib.pyplot as plt\n",
+    "from sklearn.metrics import f1_score, accuracy_score, confusion_matrix, classification_report\n",
+    "import seaborn as sns\n",
+    "\n",
+    "pd.options.display.max_rows = 200"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ddc8efe7-340b-4a6e-acf4-bb3bc8415f53",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# papermill parameters\n",
+    "report_name = None\n",
+    "model_file = None\n",
+    "X_file = None\n",
+    "y_file = None\n",
+    "params_file = None"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "027c197c-26c4-4e90-be14-4ac2c13d07bb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# check that we got all the inputs\n",
+    "missing = list()\n",
+    "for varname in [\n",
+    "    \"report_name\",\n",
+    "    \"model_file\",\n",
+    "    \"X_file\",\n",
+    "    \"y_file\",\n",
+    "    \"params_file\",\n",
+    "]:\n",
+    "    if locals()[varname] is None:\n",
+    "        missing.append(varname)\n",
+    "\n",
+    "if len(missing) > 0:\n",
+    "    raise ValueError(f\"Following inputs missing:\\n{(os.linesep).join(missing)}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "8b8a050e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load model, data and params\n",
+    "xgb_classifier = joblib.load(model_file)\n",
+    "X_test = pd.read_parquet(X_file)\n",
+    "y_test = pd.read_parquet(y_file)\n",
+    "params = None\n",
+    "with open(params_file, 'r') as f:\n",
+    "    params = json.load(f)\n",
+    "\n",
+    "# for debug:\n",
```

Comment:
I'd remove this part, or at least remove the specific file names and paths

Reply:
done